reactioncommerce/reaction-admin
# Add full GQL support to product editor

Resolves #168 #184 #212 #216 #220 #223
Impact: **major**  
Type: **feature|refactor**

## Issue

There are many issues with the current meteor specific implementation of the product admin.

## Solution

- Fix all of those problems and convert it to GraphQL
- Removed the template field (it's not available in the API in 3.0)
- Removed the google plus message field from the social for.

## Breaking changes

none.

## Testing

1. Use api with this PR: https://github.com/reactioncommerce/reaction/pull/6141
1. Use the admin and make sure everything works in the product editor

(non-critical issues can go into a list and I can address them in another PR.)
